src/main/webapp/3Dscript.js

```diff
@@ -194,8 +199,39 @@ function gui() {
   gui.add(poseTransform, 'scale', .5, 2, .25)
       .onChange(plotOrientation)
       .onFinishChange(plotTrajectory).name('Pose Scale Multiplier');
```

Comment:
Change this into a function, consider maybe separating into an array or some more condensed way to express, lots of the same gui.add line.

Reply:
i put the data point info in the for loop but not the  stuff dealing with scale/translating entire trajectory